samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -369,6 +438,50 @@ CONFIG_TEMP_TRACKING - Track temperature data
    Enables tracking and reporting of temperature data to `nRF Cloud`_.
    Defaults to enabled.
 
+
+.. _CONFIG_LED_INDICATION_PWM:
+
+CONFIG_LED_INDICATION_PWM - PWM LED indication
+   Use the `Zephyr pwm-leds`_ driver for LED indication.
+   Defaults to enabled on the Thingy:91.
+
+.. _CONFIG_LED_INDICATION_GPIO:
+
+CONFIG_LED_INDICATION_GPIO - GPIO LED indication
+   Use the `Zephyr gpio-leds`_ driver for LED indication.
+   Defaults to enabled if there is a compatible devicetree entry, and the Thingy:91 is not being targeted.
+   Defaults to enabled on the nRF9160DK.
+
+.. _CONFIG_LED_INDICATION_DISABLED:
+
+CONFIG_LED_INDICATION_DISABLED - Completely disable LED indication
+   Defaults to enabled if both :ref:`CONFIG_LED_INDICATION_PWM <CONFIG_LED_INDICATION_PWM>` and :ref:`CONFIG_LED_INDICATION_GPIO <CONFIG_LED_INDICATION_GPIO>` are disabled.
+
+.. _CONFIG_LED_INDICATOR_RGB:
+
+CONFIG_LED_INDICATOR_RGB - RGB LED Status Indication
+   Use an on-board RGB LED for status indication.
+   Defaults to enabled on the Thingy:91.
+
+.. _CONFIG_LED_INDICATOR_4LED:
+
+CONFIG_LED_INDICATOR_4LED - 4-LED Status Indication
```

Comment:
```suggestion
CONFIG_LED_INDICATOR_4LED - four-LED status indication
```

Reply:
I think "four" should be capitalized, here, since it is the start of a heading, right?

Otherwise, applied this change

Reply:
Absolutely capitalized, my bad.